pages/[[server]]/@[account]/index/index.vue

```diff
@@ -21,11 +21,12 @@ if (account) {
     title: () => `${t('account.posts')} | ${getDisplayName(account)} (@${account.acct})`,
   })
 }
+const stream = $(useStreaming(client => client.v1.stream.streamUser()))
```

Comment:
This should use the new API now, as we merged masto.js v6. See
- https://github.com/elk-zone/elk/pull/2530
```suggestion
const stream = useStreaming(client => client.user.subscribe()))
```